integration-tests/src/main/java/io/quarkus/it/hazelcast/client/IdentifiedDataSerializableWrapperFactory.java

```diff
@@ -0,0 +1,14 @@
+package io.quarkus.it.hazelcast.client;
+
+import com.hazelcast.nio.serialization.IdentifiedDataSerializable;
+
+class IdentifiedDataSerializableWrapperFactory implements com.hazelcast.nio.serialization.DataSerializableFactory {
+    @Override
+    public IdentifiedDataSerializable create(int typeId) {
+        if (typeId == 42) {
```

Comment:
If this `42` is same like in `IdentifiedDataSerializableWrapper` than this should be extracted to some constant and referred in both places.